week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr[0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr.pop();// complete this statement
```

Comment:
This does return the last element, but read the documentation closely! It will actually end up modifying arr by removing the last element from it. That's not what I would expect to happen from calling `last(arr)`. How can we return the last element without modifying arr?